BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -215,6 +215,9 @@ private void ConvertButtonTargetsRecursive(Entity button, Entity entity, float d
 			var targets = GetTargetEntities(entity);
 			foreach (var target in targets)
 			{
+				if (entity == target)
+					return;
```

Comment:
This should probably be `continue` instead. You want to avoid circular references but still check for other entities that are valid targets.